src/components/ProductItem/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Fragment } from 'react';
+import { ListIcon, ListItem, Text } from '@chakra-ui/react';
+import { FaPlus } from 'react-icons/fa';
+
+
+type ProductItemProps = {
+    items: Array<string>;
+}
+
+export const ProductItem = ({ items }: ProductItemProps) => {
+    return (
+        <Fragment>
```

Comment:
Fragment has a shorter way to use, change from

```jsx
<Fragment>
</Fragment>
```

to

```jsx
<>
</>
```

It is important to remember to close the tag the same way it was opened.

```suggestion
        <>
```

[Official doc](https://legacy.reactjs.org/docs/fragments.html#short-syntax)